packages/core/src/components/card/card.scss

```diff
@@ -77,8 +77,8 @@
   }
 
   slot[name='body']::slotted(*) {
-    font: var(--tds-detail-03);
-    letter-spacing: var(--tds-detail-03-ls);
+    font: var(--tds-detail-03) !important;
```

Comment:
I feel that using `!important` is a bit scary. But if it is the only way to fix this, then maybe have a comment about why doing it like this? Or potentially, but not part of this pr, look into the css reset we have and see if it can be replaced with something else that does not require the usage of `!important`

Reply:
Agree, the `!important` seems to be necessary due to the weakness of the `::slotted` pseudo selector. But I will add comments like you suggested 🚀